exonum/src/messages/authorization.rs

```diff
@@ -144,7 +144,7 @@ impl SignedMessage {
         public_key: &PublicKey,
     ) -> Result<(), Error> {
         if !crypto::verify(signature, &full_buffer, &public_key) {
-            bail!("Can't verify message.");
+            bail!("Can not verify message.");
```

Comment:
One-word _cannot_ is much more prevalent when it expresses inability.

Why change it anyway, though? Does that single quote break some tests or something? :)

Reply:
Because `'` is a part of a quotes and sometimes breaks parsers.

Reply:
(IMO, low-quality parsers deserve it.)